src/cmd/ui.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package cmd contains the CLI commands for UDS.
+package cmd
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/defenseunicorns/uds-cli/src/config/lang"
+	"github.com/spf13/cobra"
+)
+
+//go:embed bin/uds-runtime-*
+var embeddedFiles embed.FS
+
+var uiCmd = &cobra.Command{
+	Use:     "ui",
+	Aliases: []string{"u"},
+	Short:   lang.CmdUIShort,
+	RunE: func(_ *cobra.Command, _ []string) error {
+
+		// Create a temporary directory to hold the embedded files
+		tmpDir, err := os.MkdirTemp("", "uds-runtime-*")
+		if err != nil {
+			return fmt.Errorf("failed to create temp directory: %v", err)
+		}
+		defer os.RemoveAll(tmpDir)
+
+		var runtimeBinaryPath string
+		// Walk through the embedded files and write them to the temporary directory, eventhough we only expect one file
+		// to be embedded, the name of the binary is based on the architecture and os
+		err = fs.WalkDir(embeddedFiles, "bin", func(path string, d fs.DirEntry, err error) error {
```

Comment:
instead of walking through the dir, can we get the arch and OS via `runtime.GOOS` and `runtime.GOARCH`?

Reply:
ooh, yeah!